src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
Sure, my point is that this shouldn't be in a separate `TestRotations` test, but it the `TestQuaternion` and `TestRotationMatrix` in the `testRotate` method